types/aws-lambda/test/api-gateway-tests.ts

```diff
@@ -201,7 +198,7 @@ const authorizer: APIGatewayAuthorizerHandler = async (event, context, callback)
         str = event.resource; // $ExpectError
     } else {
         event.type; // $ExpectType "REQUEST"
-        str = event.methodArn; // $ExpectError
+        str = event.methodArn;
```

Comment:
Could you instead $ExpectError on authorizationToken, to check the condition is correctly narrowing, and move the methodArn out of the if?

Reply:
updated